packages/commitlint-config-jira/src/ingores/releases.ts

```diff
@@ -0,0 +1,5 @@
+import { Matcher } from '@commitlint/types';
+
+const RELEASE_REGEX = new RegExp('Releas(e|ing) v?\\d+\\.\\d+\\.\\d+');
+
+export const releases: Matcher = (commit: string) => RELEASE_REGEX.test(commit);
```

Comment:
By default `commitlint` ignores validation on [semver commits](https://github.com/conventional-changelog/commitlint/blob/master/%40commitlint/is-ignored/src/defaults.ts#L23), but validates commits with prefixes. 

Also, I caught that [this test case](https://github.com/bigcommerce/validate-commits/blob/14e916f0300592fc28609cfddcd7e08054c19fdc/test/commit-validator.spec.js#L56) will be valid now since they check for semver commits. IMO it's not a big deal for this one case, and it's not a breaking change, so I'm fine allowing it. If we need to still cover that, I'll try and think of a way.